user-guides/README.md

```diff
@@ -0,0 +1,16 @@
+---
+description: Angle App user guide
+---
+
+# 📔 App User Guide
+
+## Mainnet
```

Comment:
No need for the ##mainnet header here, and add a hint that the link to the app is rinkeby.angle.money